oqs_scripts/testrun.py

```diff
@@ -0,0 +1,38 @@
+import json
+import sys
+import subprocess
+import os
+import time
+
+# Parameter checks already done in shellscript
+
+with open("assignments.json", "r") as f:
+   jsoncontents = f.read();
+
+onlysigoutput = False
+if len(sys.argv)>1:
+   onlysigoutput=True
+
+assignments = json.loads(jsoncontents)
+for sig in assignments:
```

Comment:
Rather than trying to go through every algorithm, can we assemble supported KEM and signature lists as done [here](https://github.com/open-quantum-safe/boringssl/blob/master/oqs_try_handshake.py#L11), retrieve `assignments.json` and only test KEM-Signature pairs supported by BoringSSL?

Reply:
It seems like the point of this script is to verify that whatever is in BoringSSL works with the interop server. Since none of the algorithms are disabled by default, why not just list them all and only test those? That would save some time and the output would be easier to understand (if the script exits with 0 errors/failures then everything in BoringSSL still works, otherwise there's something in BoringSSL that doesn't).

Reply:
Agreed. So now implemented. Suggest to close this topic.